vmm/src/device_manager.rs

```diff
@@ -3033,7 +3054,11 @@ impl DeviceManager {
                 vfio_pci_device
                     .lock()
                     .unwrap()
-                    .update_memory(_new_region)
+                    .dma_map(
+                        _new_region.start_addr().raw_value(),
```

Comment:
Why has `_new_region` got  an `_`. It appears to be always used in this function.

Reply:
I think it is legacy from the time aarch64 didn't support PCI. I removed the `_`.